maven-scm-api/src/main/java/org/apache/maven/scm/ChangeSet.java

```diff
@@ -466,6 +518,7 @@ public String toString()
     {
         StringBuilder result = new StringBuilder( author == null ? " null " : author );
         result.append( "\n" ).append( date == null ? "null " : date.toString() ).append( "\n" );
+        result.append( "tags:" ).append( getTags() ).append( "\n" );
```

Comment:
Leave them out when they are empty as we do with other elements.

Reply:
Fixed.
I also added similar code to the toXML method.